Makefile

```diff
@@ -12,7 +12,7 @@ all:
 
 .PHONY: install
 install:
-	pip install -e .
+	$(CONDA_PREFIX)/bin/pip install -e .
```

Comment:
You cannot force a python package to be install as a conda package. This will bring mayhem to people's machines.